### PR TITLE
Allow user to view all movies

### PR DESCRIPTION
**Pull request by jamiefrancisco**
What kind of change does this PR introduce?
[] Bug fix (non-breaking change which fixes an issue)
[x] New feature (non-breaking change which adds functionality)
[] Breaking change (fix or feature that would cause existing functionality to change)

Issue/feature/bug description:
- MVP of first iteration. User is able to view all movies.
- Created mock data file
- Created Card & Movie components
- Added basic CSS

Checklist before requesting a review
[x] I have performed a self-review of my code
[x] I have refactored my code before this push
[x] The commit message follows our guidelines
[x] My code follows the code style of this project.

### CODE REVIEW

**Comment by FlorenzoBauer** [comment]
Looks Great!

File: src/components/App/App.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import './App.css';
+import  { useState } from 'react';
+import  { movieData } from '../../mockData';
+import Movies from '../Movies/Movies.js'
+
+
+function App() {
+  
+const [ movies, setMovies ] = useState(movieData.movies)
+console.log(movies);
```
Comment (kaylagordon) [inline src/components/App/App.js:10]: Remove console logs before pushing up to the main branch.

File: src/components/Card/Card.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import './Card.css'
+
+const Card = ({ average_rating, id, poster_path, title}) => {
```
Comment (kaylagordon) [inline src/components/Card/Card.js:3]: Nice job destructuring!

File: src/components/App/App.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import './App.css';
+import  { useState } from 'react';
+import  { movieData } from '../../mockData';
+import Movies from '../Movies/Movies.js'
+
+
+function App() {
+  
+const [ movies, setMovies ] = useState(movieData.movies)
```
Comment (kaylagordon) [inline src/components/App/App.js:9]: Will be changed once the fetch is implemented

File: src/components/App/App.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+console.log(movies);
+
+return (
+    <main>
+    <Movies movies={movies}/>
```
Comment (kaylagordon) [inline src/components/App/App.js:14]: indent nested elements:

```
    <main>
      <Movies movies={movies}/>
    <main>
```

**Review by kaylagordon — COMMENTED** [review]
Looks great so far!

**Pull request merged**